HW_3/task2.py

```diff
@@ -0,0 +1,12 @@
+# 2. Дан список повторяющихся элементов.
+#    Вернуть список с дублирующимися элементами.
+#    В результирующем списке не должно быть дубликатов.
+
+my_list = [3, 15, 22, 3, 3, 6, 8, 15, 9, 22]
+res_list = []
+
+for i in my_list:
```

Comment:
Тут еще можно сделать так. чтобы избежать условия в 10 строке и уменьшить количество итераций
```suggestion
for i in set(my_list):
```